CHANGELOG.md

```diff
@@ -7,10 +7,16 @@
   </summary>
 
 ### Minor
+
 * Link: Remove duplicate CSS declaration (#118)
 
 ### Patch
+
 * Docs: Updated Masonry "comp" definition to be more descriptive
+* Fixed a subtle bug in throttle that would cause longer than intended delays
```

Comment:
@btraut Could we add the PR at the end and always start the comment with a component?

Reply:
Oh, sure. Will change.